src/snowflake/snowpark/table_function.py

```diff
@@ -106,6 +108,22 @@ def over(
             new_table_function._order_by = order_spec
         return new_table_function
 
+    def alias(self, *aliases: str) -> "TableFunctionCall":
```

Comment:
Can table function itself has alias like table? If that is the case, this name might be confusing I think? Should we name it as `col_alias` for example?

Reply:
I don't think we can. The code at present does not support the table function to be renamed. I also don't see a need to rename a table function because we cal always assign a new variable to point to an existing table function, right?

Reply:
Should we also consider adding `as_` as what we did in column.py?